modules/_labs/pagination/react/lib/Pagination/common/utils/safelySpreadProps.ts

```diff
@@ -0,0 +1,20 @@
+import {flex} from './flex';
+import {layout} from './layout';
+import {space} from './space';
+
+const styleProps = {
+  ...flex,
+  ...layout,
+  ...space,
+};
+
+// TODO: Improve this function
+export function safelySpreadProps<T extends object>(props: T) {
```

Comment:
What does this function do? Is it supposed to only spread style props? There's no JSDoc or tests for it, so I can guess based on the implementation

Reply:
* Added JSDoc
* Added tests